Update readme to better reflect open docs (#22)
zed-industries/docs: README.md

# Zed Docs
Welcome to the Zed Docs repository! 

> We highly encourage contributions from the community to improve and expand the documentation. Zed is a high-performance, multiplayer code editor from the creators of Atom and Tree-sitter.. Our goal is to build the world's best text editor for developers and their teams.

Reviewer: ```suggestion
We highly encourage contributions from the community to improve and expand the documentation. Zed is a high-performance, multiplayer code editor from the creators of Atom and Tree-sitter. Our goal is to build the world's best text editor for developers and their teams.
```

## Contributing
Your contributions are invaluable to us, and we appreciate your interest in helping us improve the Zed documentation. Whether you want to fix typos, improve explanations, or add new content, every contribution is welcome. 

**Just follow these steps**:

1. Fork this repository
2. Create a new branch with a descriptive and meaningful name
3. Make your changes
4. Submit a pull request against `main`

## Structure
The docs are written in Markdown. Each file corresponds to a page in the documentation. The documentation is hosted on [GitBook](https://www.gitbook.com/) and can be accessed at [docs.zed.dev](https://docs.zed.dev/general/readme). The SUMMARY.md file in this repository defines the structure of the documentation.

### Previewing Changes
Currently, there is no way to build the documentation locally and preview your changes locally. However, if you stick to simple Markdown, most things should render correctly. 

> You can Github’s preview or a tool like [markdown live preview](https://markdownlivepreview.com/) to preview your changes.

Reviewer: ```suggestion
You can use GitHub's preview or a tool like [markdown live preview](https://markdownlivepreview.com/) to preview your changes.
```

# Zed Docs

Welcome to the Zed Docs repository!

Zed is a high-performance, multiplayer code editor from the creators of Atom and Tree-sitter. Our goal is to build the world's best text editor for developers and their teams.

We highly encourage contributions from the community to improve and expand the documentation.

## Contributing

Your contributions are invaluable to us, and we appreciate your interest in helping us improve the Zed documentation. Whether you want to fix typos, improve explanations, or add new content, every contribution is welcome.

> For more on contributing see [CONTRIBUTING.md](https://github.com/zed-industries/docs/blob/main/CONTRIBUTING.md)

Reviewer 2: minor nitpick: comma after "contributing"

### Previewing Changes

You can use Github’s preview or a tool like [markdown live preview](https://markdownlivepreview.com/) to preview your changes.

We will review your changes before merging them to ensure they look correct.

### GitBook Content Blocks and Assets

GitBook offers proprietary content blocks not expressed in plain Markdown. You can read more about [Gitbook Blocks](https://docs.gitbook.com/content-creation/blocks) on their site or by creating a free GitBook account and experimenting with your own repository.

If you need to add images or other assets to the documentation, please add them to the .gitbook/assets directory and follow the best practices outlined in the [contributing guidelines](https://github.com/zed-industries/docs/blob/main/CONTRIBUTING.md).

### Updating the Table of Contents

When adding a new file or proposing changes to the structure of the documentation, please edit the SUMMARY.md file, which serves as the table of contents for the documentation.

## Join the Zed Community

We're on a mission to create a better way to write software, and this is just the beginning.

> By contributing to the Zed Docs you're joining a community of passionate developers and helping to shape the future of software development.

Reviewer 2: minor nitpick: comma after Docs

> You can join our [discord](https://discord.gg/SSD9eJrn6s) to discuss docs, zed, any anything else text editing.

Reviewer 2: one more thing I didn't catch earlier —
```suggestion
You can join our [Discord](https://discord.gg/SSD9eJrn6s) to discuss docs, Zed, and all things text editing.
```
